src/main.cpp

```diff
@@ -188,6 +193,77 @@ void vcMain_PresentationMode(vcState *pProgramState)
     pProgramState->lastEventTime = udGetEpochSecsUTCd();
 }
 
+bool vcMain_TakeScreenshot(vcState* pProgramState, const char* pFilename = nullptr)
```

Comment:
Most of this function should be `vcTexture_SaveTexture(vcTexture *pTexture, const char *pFilename);` or `vcFrameBuffer_Save...`

Where do you prefer, @peterAdams1014?

Reply:
I've moved it to vcTexture, as the framebuffer isn't even entirely necessary except on opengl